### ISSUE TITLE
Taxonomic identification by `team`?

### ISSUE THREAD

**Issue by damianooldoni** [riparias/vmm-macrophytes-occurrences#16]
I mapped the field `team` from the sheet `meetplaatsen` as the DwC term `identifiedBy` in `occurrences.csv` as I suppose the team responsible of a place will be also the one identifying the species. 
Question for VMM: is this true?

**Comment by peterdesmet** [comment]
Also, if that is the case, shouldn't we use it for `recordedBy` as well?

**Comment by damianooldoni** [comment]
Good catch!The team is first of all the observer/recorder, so it should be coupled to the GBIF field `recordedBy` too.

**Comment by peterdesmet** [comment]
Now mapped, but not entirely sure this field is relevant.
